messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/MessagingSubsystemParser_1_0.java

```diff
@@ -261,41 +261,43 @@
                                                 .addChild(
                                                         builder(SecurityRoleDefinition.INSTANCE)
                                                                 .addAttributes(
-                                                                        SecurityRoleDefinition.SEND,
-                                                                        SecurityRoleDefinition.CONSUME,
-                                                                        SecurityRoleDefinition.CREATE_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.DELETE_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.CREATE_NON_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.MANAGE,
                                                                         SecurityRoleDefinition.DELETE_NON_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.MANAGE)))
+                                                                        SecurityRoleDefinition.CREATE_NON_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.DELETE_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.CREATE_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.CONSUME,
+                                                                        SecurityRoleDefinition.SEND
```

Comment:
This seems like a bug in WFCORE. It looks like you are adding the attributes in the opposite order as how they appear in the xsd. If a dev has to do that to get the correct output order, that seems really broken.